specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/definitions.json

```diff
@@ -3232,21 +3232,26 @@
       ],
       "description": "Policy contract Properties."
     },
-    "PolicySnippetContract": {
+    "PolicyDescriptionContract": {
       "properties": {
-        "name": {
-          "type": "string",
-          "description": "Snippet name.",
-          "readOnly": true
-        },
-        "content": {
-          "type": "string",
-          "description": "Snippet content.",
-          "readOnly": true
-        },
-        "toolTip": {
+        "properties": {
```

Comment:
I am not sure.. why the validation is not failing. But based on the example, this does not have the `properties`.